src/renderer/families/tron/Votes/index.js

```diff
@@ -51,6 +51,19 @@ const useTronSuperRepresentatives = () => {
   return sp;
 };
 
+export const useFormattedVotes = (
+  votes: ?Array<any>,
+  superRepresentatives: ?Array<any>,
+): Array<any> => {
+  return votes
```

Comment:
this could be useMemo ed?

Reply:
I guess but this is more of a data formatter than a provider I'll rename this function sorry 😅